assets/index.css

```diff
@@ -0,0 +1,1107 @@
+    display: none;
+  }
+
+  .header-login {
+    display: flex;
+  }
+
+  .header-menu {
+    padding-top: 0;
+  }
+
+  main {
+    display: grid;
+    grid-template-columns: 45% 55%;
+    grid-auto-rows: minmax(auto, auto);
+  }
+
+  .top-story-container {
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 1;
+  }
+
+  .featured-stories-container {
+    grid-column: 1;
+    grid-row: 1/5;
+    margin-top: 0;
+  }
+
+  .turn-container {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 2;
+    margin-top: 0 !important;
+  }
+
+  .culture-travel-container {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
+    grid-column: 2;
+    grid-row: 3;
+    margin-top: 0 !important;
+  }
+
+  .more-stories-container {
+    padding-top: 0 !important;
+    overflow-y: scroll;
+    height: 79.5em;
+    margin-top: 0 !important;
+    grid-row: 4;
+    grid-column: 2;
+  }
+
+  .debate-container {
+    justify-content: center;
+    margin-top: 0 !important;
+    grid-row: 5;
+    grid-column: 1 / 3;
+  }
+
+  .opinion-container {
+    margin-top: 0 !important;
+    grid-row: 6;
+    grid-column: 1 / 3;
+  }
+
+  .newsletter-signup-container {
+    grid-column: 1 / 3;
+    grid-row: 7;
+  }
+
+  .magazine {
+    margin-top: 0 !important;
+    grid-column: 1 / 3;
+    grid-row: 8;
+    padding-bottom: 0 !important;
+  }
+
+  .editors-pick {
+    margin-top: 0 !important;
+    grid-column: 1 / 3;
+    grid-row: 9;
+    padding: 0 !important;
+  }
+
+  .picks-container {
+    grid-column: 1 / 3;
+    grid-row: 10;
+  }
+
+  .bottom-page {
+    grid-column: 1 / 3;
+    grid-row: 11;
+  }
+
+  footer {
+    grid-column: 1 / 3;
+    grid-row: 12;
+  }
+
+  .more-stories-container::-webkit-scrollbar {
+    display: none;
+  }
+
+  .main-featured-stories {
+    max-height: 21em;
+  }
+
+  .main-top-story {
+    max-height: 24em;
+  }
+
+  .item-turn1,
+  .cult-item1 {
+    grid-column: 1;
+  }
+
+  .item-turn2,
+  .cult-item2 {
+    grid-column: 2;
+  }
+
+  .item-turn3,
+  .cult-item3 {
+    grid-column: 3;
+  }
+
+  .images-container {
+    display: grid !important;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+  }
+
+  .image-turn,
+  .image-culture {
+    width: 15vw;
+    height: 14vw;
+  }
+
+  .subt-turn,
+  .subt-culture {
+    font-size: 1.5vw;
+  }
+
+  .mag-pick-cont {
+    width: 23%;
+  }
+
+  .cover-image-magazine {
+    width: 22vw;
+    height: 30vw;
+  }
+
+  .image-magazine {
+    height: 20em;
+  }
+
+  .mag-item-cont {
+    padding-top: 2vw;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .picked-cont {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .picked-item-cont {
+    width: 50%;
+    padding: 1.5vw;
+  }
+
+  .editors-item {
+    height: 22em;
+  }
+
+  .picks-item {
+    flex-direction: row !important;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
```

Comment:
You can implement flex-direction with bootstrap. Please see this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information

Reply:
IF you check my HTML you can find the flex-direction row class in several elements